src/components/wallet/ImportPhrase.tsx

```diff
@@ -94,36 +93,33 @@ const ImportPhrase: React.FC = (): JSX.Element => {
   return (
     <>
       {renderError}
-      <Form form={form} onFinish={submitForm} labelCol={{ span: 24 }}>
+      <Form form={form} onFinish={submitForm} labelCol={{ span: 24 }} style={{ padding: 30, paddingTop: 15 }}>
         {/* TODO(@Veado): i18n */}
         <Spin spinning={importing} tip="Loading...">
           <Form.Item
             name="phrase"
             rules={[{ required: true, validator: phraseValidator }]}
             validateTrigger={['onSubmit', 'onChange']}>
             {/* TODO(@Veado): i18n */}
-            <Input.Password placeholder="Enter your phrase" size="large" />
+            <InputTextArea placeholder="ENTER PHRASE" rows={5} />
```

Comment:
Add any text values into our `i18n` sources. Please do the same for all other texts in this PR.